appveyor.yml

```diff
@@ -1,48 +1,15 @@
-# AppVeyor.com is a Continuous Integration service to build and run tests under
-# Windows
-
-environment:
-
-  global:
-      PYTHON: "C:\\Miniconda36-x64"
-
-  matrix:
-      - PYTHON_VERSION: "3.8"
+# This is a dummy configuration file that *does not run any test*.
+# The file should be deleted, and Appveyor deactivated as soon as all
+# opened PRs have this configuration merged in.
+# SEE PR#2908 (https://github.com/yt-project/yt/pull/2908/files)
 
 platform:
     - x64
 
 install:
-    - "if not exist \"%userprofile%\\.config\\yt\" mkdir %userprofile%\\.config\\yt"
-    - "echo [yt] > %userprofile%\\.config\\yt\\ytrc"
-    - "echo suppressStreamLogging = True >> %userprofile%\\.config\\yt\\ytrc"
-    - "SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%"
-    - "copy tests\\matplotlibrc ."
-
-    # Install the build and runtime dependencies of the project.
-    # Create a conda environment
-    - "conda update -q --yes conda"
-    - "conda create -q --yes -n test python=%PYTHON_VERSION%"
-    - "activate test"
+    - "echo 'Not installing anything.'"
```

Comment:
These are not the droids you're looking for.

Reply:
Damned, I missed a really good opportunity.

Reply:
Good one,
> I'm sorry Dave, I'm afraid I cannot do that

takes all the love, to the point it's almost annoying sometimes :)